ts/src/bitget.ts

```diff
@@ -3339,22 +3342,37 @@ export default class bitget extends Exchange {
             const side = this.safeString (rawOrder, 'side');
             const amount = this.safeValue (rawOrder, 'amount');
             const price = this.safeValue (rawOrder, 'price');
-            const orderParams = this.safeValue (rawOrder, 'params', {});
+            let orderParams = this.safeValue (rawOrder, 'params', {});
+            [ marginMode, orderParams ] = this.handleMarginModeAndParams ('createOrders', params);
```

Comment:
@Dan-krm we can call `handleMarginModeAndParams` and discard the omitted `params` so we don't need to re-assign it again, by doing:

```Typescript
const marginResult = this.handleMarginModeAndParams ('createOrders', params);
const marginMode = marginResult[0]
```

Reply:
Also similar to what we do to the symbol, we should check if the marginMode is the same on every order